spec/features/user_searches_places_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+feature 'A user searches places' do
+  scenario 'and views matching places' do
+    irrelevant_places = create_list(:place, 5, :active, name: 'Playmoweb')
```

Comment:
Useless assignment to variable - `irrelevant_places`. Did you mean `relevant_places`?